coral-component-taglist/src/scripts/TagList.js

```diff
@@ -312,6 +312,11 @@ class TagList extends BaseFormField(BaseComponent(HTMLElement)) {
     detachedItem.removeAttribute('tabindex');
     detachedItem._host = undefined;
     
+    //if all tags are removed focus should move to previously focused element i.e parent element 
+    if (this._itemToFocusAfterDelete && this.parentElement && (detachedItem === this._itemToFocusAfterDelete)) {
```

Comment:
@majornista Is this really always parent element ? We could also store the previously focused element and restore it wdyt ?

Reply:
@icaraps @majornista : I have incorporated changes